docs/data/base/components/progress/UnstyledProgressIntroduction.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { useTheme } from '@mui/system';
+import * as Progress from '@base_ui/react/Progress';
+
+export default function UnstyledProgressIntroduction() {
+  return (
+    <div className="App">
+      <Progress.Root className="Progress" value={50} aria-labelledby="ProgressLabel">
+        <span className="Progress-label" id="ProgressLabel">
+          Uploading files
+        </span>
+        <Progress.Track className="Progress-track">
+          <Progress.Indicator className="Progress-indicator" />
+        </Progress.Track>
+      </Progress.Root>
+      <Styles />
+    </div>
+  );
+}
+
+function useIsDarkMode() {
+  const theme = useTheme();
+  return theme.palette.mode === 'dark';
+}
+
+export function Styles() {
+  const isDarkMode = useIsDarkMode();
+  return (
+    <style suppressHydrationWarning>{`
```

Comment:
What this suppression for? If there are differences due to encoding of some characters, try <style dangerouslySetInnerHTML={{ __html: '' }} />

Reply:
It was for pseudo-elements ~ but I've removed it as I ended up not using any

Reply:
It was for some encoding issue, but I've fixed the root cause now 😅